Source/ZoomNet/Models/PaginatedResponseWithTokenAndDateRange.cs

```diff
@@ -0,0 +1,58 @@
+using Newtonsoft.Json;
+
+namespace ZoomNet.Models
+{
+	/// <summary>
+	/// Pagination Object.
+	/// </summary>
+	/// <typeparam name="T">The type of records.</typeparam>
+	public class PaginatedResponseWithTokenAndDateRange<T>
+	{
+		/// <summary>
+		/// Gets or sets the start date for the report.
+		/// </summary>
+		/// <value> Start date for this report in 'yyyy-mm-dd' format.</value>
+		public string From { get; set; }
+
+		/// <summary>
+		/// Gets or sets the end date for the report.
+		/// </summary>
+		/// <value> End date for this report in 'yyyy-mm-dd' format.</value>
+		public string To { get; set; }
```

Comment:
Type should be `DateTime`
```suggestion
		public DateTime To { get; set; }
```